src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationManagementTest.java

```diff
@@ -643,10 +643,18 @@ void testGenerateArtefactSummaryWpafccWeeklyHearingList() throws Exception {
         assertTrue(responseContent.contains(CASE_REFERENCE_NUMBER_FIELD), CONTENT_MISMATCH_ERROR);
     }
 
-    @Test
-    void testGenerateArtefactSummaryUtIacJudicialReviewDailyHearingList() throws Exception {
+    @ParameterizedTest
+    @EnumSource(
+        value = ListType.class,
+        names = {
+            "UT_IAC_JR_LONDON_DAILY_HEARING_LIST",
+            "UT_IAC_JR_MANCHESTER_DAILY_HEARING_LIST",
+            "UT_IAC_JR_BIRMINGHAM_DAILY_HEARING_LIST",
+            "UT_IAC_JR_CARDIFF_DAILY_HEARING_LIST"
+        })
+    void testGenerateArtefactSummaryUtIacJudicialReviewLondonDailyHearingList(ListType listType) throws Exception {
```

Comment:
The test method name does not need to mention London. This test is testing all 4 regions

Reply:
Fixed.